x-pack/test/api_integration/apis/actions/builtin_action_types/slack.ts

```diff
@@ -8,12 +8,26 @@ import expect from '@kbn/expect';
 
 import { KibanaFunctionalTestDefaultProviders } from '../../../../types/providers';
 
+import { SLACK_ACTION_SIMULATOR_URI } from '../../../fixtures/plugins/actions';
+
+// const xPackAPITestsConfig = await readConfigFile(require.resolve('../api_integration/config.js'));
```

Comment:
why is this commented out code still here?

Reply:
removed the commented out code